tests/isisd/test_isis_vertex_queue.c

```diff
@@ -16,43 +16,42 @@ static size_t vertex_count;
 
 static void setup_test_vertices(void)
 {
-	struct prefix p = {
-		.family = AF_UNSPEC
+	union isis_N nid, nip = {
+		.prefix.family = AF_UNSPEC
 	};
-	uint8_t node_id[7];
 
 	vertices = XMALLOC(MTYPE_TMP, sizeof(*vertices) * 16);
 
-	p.family = AF_INET;
-	p.prefixlen = 24;
-	inet_pton(AF_INET, "192.168.1.0", &p.u.prefix4);
-	vertices[vertex_count] = isis_vertex_new(&p, VTYPE_IPREACH_TE);
+	nip.prefix.family = AF_INET;
+	nip.prefix.prefixlen = 24;
+	inet_pton(AF_INET, "192.168.1.0", &nip.prefix.u.prefix4);
+	vertices[vertex_count] = isis_vertex_new(&nip, VTYPE_IPREACH_TE);
```

Comment:
Should this be `&nip.prefix`

Reply:
@qlyoung Did you mean &nip.prefix.u? (&nip.prefix targets the first element of "struct prefix" -uint8_t family-, while prefix data is in the 3rd element of that struct -union {} u)

Reply:
I read a little deeper into the code and it looks fine. Thanks for following up.